ansible/roles/base/tasks/execute.yml

```diff
@@ -0,0 +1,46 @@
+- name: check if socket exists
+  stat:
+    path: /tmp/firecracker.socket
+  register: socket_exists
+
+- name: set kvm permissions
+  shell: sudo setfacl -m u:${USER}:rw /dev/kvm
+
+- name: connect to socket
+  shell: nohup src/firecracker/build/release/firecracker --api-sock /tmp/firecracker.socket </dev/null >/dev/null 2>&1 &
+
+- name: add kernel
+  shell: >
+         curl --unix-socket /tmp/firecracker.socket -i
+         -X PUT 'http://localhost/boot-source'
+         -H 'Accept: application/json'
+         -H 'Content-Type: application/json'
+         -d '{
+               "kernel_image_path": "./hello-vmlinux.bin",
+               "boot_args": "console=ttyS0 reboot=k panic=1 pci=off"
+             }'
+
+- name: add fs
+  shell: >
+         curl --unix-socket /tmp/firecracker.socket -i \
+         -X PUT 'http://localhost/drives/rootfs' \
+         -H 'Accept: application/json'           \
+         -H 'Content-Type: application/json'     \
+         -d '{
+              "drive_id": "rootfs",
+              "path_on_host": "./hello-rootfs.ext4",
```

Comment:
Is this the same path as the one above?
"dest: ~/hello-rootfs.ext4"
- Are they even ran by the same user?
- Even if so, is theh path the same?

Reply:
> Is this the same path as the one above?
yes
> Are they even ran by the same user?
yes
> Even if so, is theh path the same?
not sure what you mean, it downloads the kernel and the rootfs and the curl commands plug them to the firecracker VM

Reply:
You said it's the same path - but it's not the same path string.
One uses dot, the other tilde. If it *happens* to be the same path that's nice, but why rely on it? why not use the same path string for both? (either dot or tilde)

Reply:
Done